samples/gen-go-db/server/db/tests/tests.go

```diff
@@ -5345,6 +5356,331 @@ func ScanThingWithCompositeAttributesRepeatedsByTresAndUnoDos(d db.Interface, t
 	}
 }
 
+type getThingWithCompositeAttributesRepeatedsByCuatroAndTresInput struct {
+	ctx   context.Context
+	input db.GetThingWithCompositeAttributesRepeatedsByCuatroAndTresInput
+}
+type getThingWithCompositeAttributesRepeatedsByCuatroAndTresOutput struct {
+	thingWithCompositeAttributesRepeateds []models.ThingWithCompositeAttributesRepeated
+	err                                   error
+}
+type getThingWithCompositeAttributesRepeatedsByCuatroAndTresTest struct {
+	testName string
+	d        db.Interface
+	input    getThingWithCompositeAttributesRepeatedsByCuatroAndTresInput
+	output   getThingWithCompositeAttributesRepeatedsByCuatroAndTresOutput
+}
+
+func (g getThingWithCompositeAttributesRepeatedsByCuatroAndTresTest) run(t *testing.T) {
+	thingWithCompositeAttributesRepeateds := []models.ThingWithCompositeAttributesRepeated{}
+	fn := func(m *models.ThingWithCompositeAttributesRepeated, lastThingWithCompositeAttributesRepeated bool) bool {
+		thingWithCompositeAttributesRepeateds = append(thingWithCompositeAttributesRepeateds, *m)
+		if lastThingWithCompositeAttributesRepeated {
+			return false
+		}
+		return true
+	}
+	err := g.d.GetThingWithCompositeAttributesRepeatedsByCuatroAndTres(g.input.ctx, g.input.input, fn)
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+	require.Equal(t, g.output.err, err)
+	require.Equal(t, g.output.thingWithCompositeAttributesRepeateds, thingWithCompositeAttributesRepeateds)
+}
+
+func GetThingWithCompositeAttributesRepeatedsByCuatroAndTres(d db.Interface, t *testing.T) func(t *testing.T) {
+	return func(t *testing.T) {
+		ctx := context.Background()
+		require.Nil(t, d.SaveThingWithCompositeAttributesRepeated(ctx, models.ThingWithCompositeAttributesRepeated{
+			Cuatro: db.String("string1"),
+			Tres:   db.String("string1"),
+			Uno:    db.String("string1"),
+		}))
```

Comment:
The issue was that the created struct did not include all composite keys. When we save the fields into the DB, we build the composite keys and reference the attributes (they must not be null) in order to do so, in this case it is uno and dos, but dos is not defined